Tests/js/views/assets/ez-dashboardblocksview-tests.js

```diff
@@ -78,7 +76,7 @@ YUI.add('ez-dashboardblocksview-tests', function (Y) {
             blocks = view.get('blocks');
 
             Y.Assert.areSame(view, block1.getTargets()[0], 'The dashboard view should be a target of a block');
-            Y.Assert.areSame(1, blocks.length, 'There should be 1 block view instance available');
+            Y.Assert.areSame(2, blocks.length, 'There should be 2 block view instances available');
```

Comment:
also, you have to add a test case for the default value of `blocks`, for now you have to make sure a `eZ.DashboardBlockAllContentView` is instantiated, that the bubble target is correctly defined and once https://github.com/ezsystems/PlatformUIBundle/pull/589/files#r64005450 is taken into account that it receives the root location.